src/apyfixedarray.cc

```diff
@@ -221,7 +221,7 @@ APyFixedArray APyFixedArray::matmul(const APyFixedArray& rhs) const
             return _checked_inner_product(rhs);
         }
     }
-    if (ndim() == 2 && rhs.ndim() == 2) {
+    if ((ndim() == 2 && rhs.ndim() == 2) || (ndim() == 2 && rhs.ndim() == 1)) {
```

Comment:
```suggestion
    if (ndim() == 2 && (rhs.ndim() == 2 || rhs.ndim() == 1)) {
```
there is no "in tuple"-syntax in C++?

Reply:
Unfortunately, as far as I know, there is no `A in B` syntax in C++.